CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# v2.1.0
```

Comment:
Should this be considered a bug and thus a 2.0.1 release?

Reply:
Probably, if this will get released independently of #4.

Reply:
Switched to 2.0.1: https://github.com/cerner/cerner-oauth1a/pull/5/commits/31f56397b5e76add30320dcafa03823e2ff2efc5